packages/core/src/checkout/checkout-action-creator.ts

```diff
@@ -34,22 +34,23 @@ export default class CheckoutActionCreator {
                         params: { ...options?.params, checkoutId: id },
                     }),
                     this._formFieldsActionCreator.loadFormFields({ ...options, useCache: true }),
-                    defer(() => {
-                        const state = store.getState();
-                        const config = state.config.getStoreConfigOrThrow();
-
-                        console.log(config);
-
-                        return this._checkoutRequestSender
-                            .loadCheckout(id, options)
-                            .then(({ body }) => {
-                                return createAction(
-                                    CheckoutActionType.LoadCheckoutSucceeded,
-                                    this._transformCustomerAddresses(body),
-                                );
-                            });
-                    }),
                 ),
+                defer(() => {
```

Comment:
before the change, while manual test, I go to checkout page and it crashes:
![image](https://github.com/bigcommerce/checkout-sdk-js/assets/94653751/91f32294-a77b-4b7d-a22c-1d75c4bb879c)

I tried to make a change from
contact(of(), merge(..., ..., defer()))
to 
contact(of(), merge(..., ...), defer())

then manual test works fine. but the core test fails on one case. So I don't really know how to deal with it.